example/server.py

```diff
@@ -1,11 +1,25 @@
+import pickle
+
 from bottle import Bottle, redirect, run
+import datastore.core
 
 from switchboard import operator, configure
 from switchboard.middleware import SwitchboardMiddleware
 from switchboard.admin import app as switchboard
 
-configure()
 
+# Setup a file-based datastore with pickle serialization.
```

Comment:
I originally had these imports up in their usual place, but when I started playing around with connecting to different types of datastores (Mongo, Redis, etc.) I found it handy to have the datastore-specific imports by the code that setup the datastore. Then I could snag the imports when I commented/uncommented a block.